version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.3-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.1.4"
```

Comment:
This version file is handled by the [sbt-release](https://github.com/sbt/sbt-release) plugin. It is automatically bumped when the release command is run, and should always be set to snapshot during normal development.